src/main/java/com/ibm/crypto/plus/provider/ock/NativeInterface.java

```diff
@@ -139,7 +139,7 @@ static void preloadJGskit(String libraryToLoad) {
                 // win64_x86
                 loadFile = new File(jgskitPath, "libjgskit_64.dll");
             }
-        } else if ((osArch.equals("aarch64")) && (osName.equals("Mac OS X"))) {
+        } else if (osName.equals("Mac OS X")) {
```

Comment:
Can we also combine the logic in the code blocks in this file starting on line 271 and line 178? I dont see the need here to seperate these?

Also line 274:
```
requirePreloadOCK = false;
```
Could be set like other platforms and removed?

Reply:
Updated this line and also removed unnecessary duplicate code with 77de5f5c71354b67a3ef0f910a80a2699fceb735.

Reply:
Looks good to me.